### ISSUE TITLE
Cleanup tool/internal service for resources of crashed applications

### ISSUE THREAD

**Issue by elfenpiff** [elkodon/elkodon#65]
## Brief feature description

When an application crashes, it can happen that a service is only partially removed and becomes unusable. 
An internal service and/or a command line application shall cleanup the remainders of such services.

The following use cases shall be considered:

* remove static service info, when only the dynamic service info exists
* remove dynamic service info, when only the static service info exists
* remove the data segment of a publisher if the owning process no longer exists
* remove the publisher/subscriber as participant from the dynamic service info when the owning process no longer exists.

**Comment by elBoberido** [comment]
Could this be done on the startup of the application?

**Comment by elfenpiff** [comment]
> Could this be done on the startup of the application?

Yes. The basic idea would be to implement a "funtion" that does it in a thread-safe manner and then either everyone can do this in a decentral fashion or one central daemon or command line tool does this.

This is then up to the deployment/user.
